<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicSysClass</class>
 <widget class="QMainWindow" name="GraphicSysClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphicSys</string>
  </property>
  <widget class="QWidget" name="m_pCentralWidget"/>
  <widget class="QMenuBar" name="m_pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <widget class="QToolBar" name="m_pMainToolBar">
   <property name="windowTitle">
    <string>顶部工具栏</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="m_pStatusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GraphicSys.qrc"/>
 </resources>
 <connections/>
</ui>
